MacDown/Code/Preferences/MPPreferences.m

```diff
@@ -173,6 +175,7 @@ - (void)loadDefaultPreferences
     self.editorOnRight = NO;
     self.editorShowWordCount = NO;
     self.editorWordCountType = 0;
+    self.previewZoomRelativeToBaseFontSize = NO;
```

Comment:
This line is not needed; the line in `-loadDefaultUserDefaults` does the same thing. And even that if clause is not needed, either, since an empty preference defaults to `NO` anyway.